src/Categories/Category/Instance/FamilyOfSetoids.agda

```diff
@@ -11,13 +11,23 @@ open import Level
 open import Relation.Binary
   using (Rel; Setoid; module Setoid; Reflexive; Symmetric; Transitive)
 open import Function.Base renaming (id to idf; _∘_ to _⊚_)
-open import Function.Equality
-open import Function.Inverse using (_InverseOf_)
+open import Function.Bundles using (Func; _⟨$⟩_)
+open import Function.Construct.Identity using () renaming (function to idF)
+open import Function.Construct.Composition using (function)
+open import Function.Construct.Setoid renaming (function to func-setoid)
+open import Function.Structures using (IsInverse)
 import Relation.Binary.Reasoning.Setoid as SetoidR
 
 open import Categories.Category
 
 module _ {a b c d : Level} where
+
+  private
+    infixr 9 _∙_
+    _∙_ : {a₁ a₂ b₁ b₂ c₁ c₂ : Level} {A : Setoid a₁ a₂} {B : Setoid b₁ b₂}
```

Comment:
If we're going to use this across several modules, can we put it somewhere it can be imported?

Reply:
Done